pwa/src/layout/Layout.tsx

```diff
@@ -33,6 +34,12 @@ const Layout: React.FC<LayoutProps> = ({ children, pageContext }) => {
   return (
     <Document>
       <Page className="Page">
+        <TopNav
+          items={[
+            { href: "/", title: "Home" },
+            { href: "/meldingen/", title: "Meldingen" },
+          ]}
+        />
```

Comment:
I moved this to a Header template. That way, the `Logo` doesn't need to be within the `TopNav`. Also, I added the `Breadcrumbs` to this HeaderTemplate.